raft/raft_paper_test.go

```diff
@@ -603,14 +603,14 @@ func TestFollowerCheckMsgApp(t *testing.T) {
 	tests := []struct {
 		term        uint64
 		index       uint64
+		windex      uint64
 		wreject     bool
 		wrejectHint uint64
 	}{
-		{ents[0].Term, ents[0].Index, false, 0},
-		{ents[0].Term, ents[0].Index + 1, true, 2},
-		{ents[0].Term + 1, ents[0].Index, true, 2},
```

Comment:
could we change the `pb.HardState{Commit: 2}` to `pb.HardState{Commit: 1}` instead?